src/lib/Scenes/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -1,37 +1,64 @@
+import { Box, Join, Separator, Spacer } from "@artsy/palette"
 import { OtherWorks_artwork } from "__generated__/OtherWorks_artwork.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import { filter } from "lodash"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-import { ArtworkContextArtistFragmentContainer as ArtworkContextArtist } from "./ArtworkContexts/ArtworkContextArtist"
+import { ContextGridCTA } from "./ContextGridCTA"
+import { Header } from "./Header"
 
-export const OtherWorksFragmentContainer = createFragmentContainer<{
-  artwork: OtherWorks_artwork
-}>(
+export const populatedGrids = grids => {
```

Comment:
We could type this `grids` parameter

```suggestion
export const populatedGrids = (grids: OtherWorks_artwork['contextGrids']) => {
```